popup/translations.ts

```diff
@@ -14,18 +14,27 @@ languages
     languageSelector.appendChild(option)
   })
 
-export const translations = languages.reduce(
+export const translations: {
+  [code: string]: {
+    [key: string]: string
+  }
```

Comment:
Let's use TypeScript's utility types for records: `Record<string, Record<string, string>>`